guillotina/contentapi.py

```diff
@@ -37,6 +37,7 @@ def __init__(self, db, user=RootUser("root")):
         await self.abort()
 
     async def use_container(self, container: IContainer):
+        task_vars.registry.set(None)
```

Comment:
is the `registry` task var set on `get_registry(container)`?

Reply:
ah yeah! I see it now :+1:

Reply:
Yes, it is! https://github.com/plone/guillotina/blob/master/guillotina/utils/misc.py#L381